docs/custom-domain/registrars/cloudflare/cloudflare.md

```diff
@@ -0,0 +1,155 @@
+
+- Set the record **Type** to `TXT`
+- Add the value `@` under in the **Name** field
+- Keep **TTL** as `Auto`
+- Copy/paste the value of SimpleLogin DNS page in the **Content** field
+
+![](./cloudflare-add-domain.png)
+
+Now click on **"Verify"** on SimpleLogin, your domain ownership should be now verified. Please note that it can take up to several minutes for the DNS change to be available. In this case don't hesitate to click on **"Verify"** several minutes later.
+
+Once your domain ownership is verified, you can continue with the rest of the setup.
+
+![](../sl-domain-ownership.png)
+
+## 2. **Add MX record on a Cloudflare domain**
+
+From the [Cloudflare DNS configurations panel](/docs/custom-domain/registrars/cloudflare/cloudflare/#access-the-dns-configurations-panel-of-your-domain-on-cloudflare), click on the **“Add record”** again.
+
+**Then create a first MX record with:**
+
+- Set the record **Type** to `MX`
+- Add the value `@` under in the **Name** field
+- Add the value `mx1.simplelogin.co.` under in the **Mail server** field
+- Set **Priority** to `10`
+
+
+**(again) Add a second MX record with:**
+
+- Set the record **Type** to `MX`
+- Add the value `@` under in the **Name** field
+- Add the value `mx2.simplelogin.co.` under in the **Mail server** field
+- Set **Priority** to `20`
+
+![](./cloudflare-add-mx1.png)
+
+If you have existing MX records, you need to remove them by clicking **"Edit”** and **“Delete”.**
+
+*Here is Cloudflare’s documentation about [How to delete a DNS record on Cloudflare](https://developers.cloudflare.com/dns/manage-dns-records/how-to/create-dns-records/#delete-dns-records)*
+
+Now click on **"Verify"** on SimpleLogin, the MX records should be now verified. As it can take up to several minutes for the DNS change to be available, don't hesitate to click on **"Verify"** several minutes later.
+
+![](../domain-verified.png)
+
+Once the MX record is verified, you can start creating aliases with your domain! Click on "New Custom Alias", you should now see your domain in the list of available domains.
+
+![](../domain-added.png)
+
+## **(Optional) 3. Add TXT record on a Cloudflare domain for SPF**
+
+Setting up SPF is recommended if you plan to send emails from your aliases. It'll reduce the chance your emails ending up in the recipient's Spam folder.
+
+From the [Cloudflare DNS configurations panel](/docs/custom-domain/registrars/cloudflare/cloudflare/#access-the-dns-configurations-panel-of-your-domain-on-cloudflare), click on the **"Add record"** button. 
+
+![](./cloudflare-add-spf.png)
+
+**Then add the following TXT record:**
+
+- Set the record **Type** as `TXT`
+- Add the value `@` under in the **Name** field
+- Keep **TTL** as `Auto`
+- Copy/paste `v=spf1 include:simplelogin.co ~all` in the **Content** field
+
+
+Go back to SimpleLogin and click on **"Verify",** the SPF should be now verified.
+
+![](./cloudflare-spf-verified.png)
```

Comment:
this image doesn't exist, we can either remove it here or add the missing image.